common/landed_titles/01_landed_titles.txt

```diff
@@ -1,4 +1,4 @@
-﻿###### AUTO GENERATED CONTENT (c) CKTools2 0.1.0A ######
+﻿###### AUTO GENERATED CONTENT (c) CKTools3 1.3.1A ######
 
```

Comment:
This province is too tiny. It should be either expanded or merged with another one (but don't forget to clean up after)
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/143527069-063a374c-367d-420b-9fb8-18692d8813fd.png)

</details>

Reply:
Noted, will resolve during the full Kalimdor revamp.